src/main/java/core/basesyntax/Buldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Buldozer extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Buldozer started working");

```

Comment:
The message 'Buldozer started working' is not fully informative. It should indicate what type of machine is working, which is done, but consider specifying the action it is performing in more detail.